source/MultiGrid/MultiGrid.js

```diff
@@ -238,13 +238,15 @@ class MultiGrid extends React.PureComponent {
             ...rest,
             onScroll,
             scrollLeft,
+            isScrolling: isScrolling || undefined,
```

Comment:
Can this be just `isScrolling`?

Reply:
@wuweiweiwu,
Good question.

It can but that causes the test **should call forceUpdate() on inner Grids** to fail because there is a 5th `cellRenderer` call. I could change the test but doing an extra render would cause a bit of overhead so I choose for this solution.

Reply:
hm interesting. I would think that setting isScrolling to false should have the same effect as setting it to undefined

Reply:
Same for me but my experience was different. No harm in double checking, maybe I did something wrong, as it takes only a minimal effort.